firewood/src/v2/api.rs

```diff
@@ -82,6 +82,15 @@ pub enum Error {
 
     #[error("Range too small")]
     RangeTooSmall,
+
+    #[error("Child not found")]
+    ChildNotFound,
+
+    #[error("Extension node is at the root; should be branch node")]
```

Comment:
This is only referring to sentinel root? as his can happen for the real root, right? The sentinel root we force it to be branch node.

Reply:
updated name